task/06-conditions-n-loops-tasks.js

```diff
@@ -296,7 +349,21 @@ function getDigitalRoot(num) {
  *   '{[(<{[]}>)]}' = true 
  */
 function isBracketsBalanced(str) {
-    throw new Error('Not implemented');
+    let arr = Array(0);
+    
+    for (let el of str.split('')){
+        if (el == '(' || el == '[' || el == '{' || el == '<') {
+            arr.push(el);
+            continue;
+        }
+        if ((el == ')' && arr[arr.length - 1] == '(') || (el == '}' && arr[arr.length - 1] == '{') || 
```

Comment:
This is too cumbersome. 

Also if you'd like to add new type of bracket you'd have make changes in 2 places. Tyr to refactor this  so you 'd have to make change only in 1 place to add a new bracket